test/libsolidity/SolidityParser.cpp

```diff
@@ -543,7 +543,8 @@ BOOST_AUTO_TEST_CASE(keyword_is_reserved)
 		"supports",
 		"switch",
 		"typedef",
-		"typeof"
+		"typeof",
+		"var"
```

Comment:
Does this change have to do with checked arithmetic?

Reply:
Moved: https://github.com/ethereum/solidity/pull/9965